src/components/elements/carousel/carousel.js

```diff
@@ -36,6 +38,16 @@ PrevButton.propTypes = {
     onClick: PropTypes.func,
 }
 
+export const NavigationButton = ({ color, is_enabled, onClick }) => (
+    <StyledDot onClick={onClick} color={is_enabled ? color : 'grey'} />
```

Comment:
```suggestion
    <StyledDot onClick={onClick} color={is_enabled ? color : null} />
```
Since the default color in grey we don't need to pass `grey`